src/pages/ctrader-signup/_signup-success-form.tsx

```diff
@@ -0,0 +1,139 @@
+import React, { useEffect } from 'react'
+import { Controller } from 'react-hook-form'
+import {
+    form_style,
+    input_style,
+    dropdown_style,
+    paragraph_style,
+} from './ctrader-form.module.scss'
+import { localize, Localize } from 'components/localization'
+import { DropdownSearch } from 'components/elements'
+import Input from 'features/components/atoms/input'
+import { TString } from 'types/generics'
+import useCtraderSubmitForm from 'features/hooks/use-ctrader-submit-form'
+import Flex from 'features/components/atoms/flex-box'
+import Typography from 'features/components/atoms/typography'
+import Button from 'features/components/atoms/button'
+import { ResidenceType } from 'components/hooks/use-residence-list'
+
+type SignupSuccessFormProps = {
+    email: string
+    residence_list: ResidenceType[]
+}
+
+const SignupSuccessForm = ({ email, residence_list }: SignupSuccessFormProps) => {
+    const { submitForm, onSubmit, loading } = useCtraderSubmitForm()
+
+    const {
+        register,
+        formState: { errors, isValid },
+        watch,
+        handleSubmit,
+        clearErrors,
+        setValue,
+        setError,
+        control,
+    } = submitForm
+    const values = watch()
+
+    useEffect(() => {
+        const response_error = errors.root?.serverError
+        if (response_error?.type === 'InvalidToken') {
+            clearErrors()
+            setError('verification_code', { message: response_error.message })
+        }
+    }, [clearErrors, errors.root?.serverError, setError])
+
+    const isButtonDisabled =
+        values.verification_code === '' ||
+        values.password === '' ||
+        values.residence === '' ||
+        !isValid ||
+        loading
+
+    return (
+        <Flex.Box
+            as="form"
+            justify="center"
+            align="center"
+            direction="col"
+            gap="10x"
+            className={form_style}
+            onSubmit={handleSubmit(onSubmit)}
+        >
+            <Typography.Paragraph className={paragraph_style} align="center">
+                <Localize
+                    translate_text="_t_Please enter the 8 character verification code that was sent to {{email}} to activate your account._t_"
+                    values={{ email }}
+                />
+            </Typography.Paragraph>
+            <Input.Text
+                className={input_style}
+                autoComplete="none"
+                autoCapitalize="none"
+                id="verification_code"
+                label={'_t_Verification code_t_'}
+                clearErrors={clearErrors}
+                setValue={setValue}
+                error={
+                    errors?.verification_code?.message
+                        ? localize(errors?.verification_code?.message as TString)
```

Comment:
If this error will different for each `verification_code` localize have no sense
If this error will the same, we need to move this text outside and wrap to `_t_`

```suggestion
const error_message: TString = `_t_${errors?.verification_code?.message}_t_`

                        ? localize(error_message)
```

Reply:
We have error message for all 3 fields. Should we do this for all of them?

Reply:
yes, please

Reply:
@mohsen-deriv We already have a similar code in signup component:
`error={errors?.email?.message ? localize(errors?.email?.message as TString) : null}`
Regarding the issue that @NikitK-deriv has mentioned above, does this approach work as expected?

Reply:
@mohammad-hashemi-deriv let's do it with @NikitK-deriv's approach, thanks :pray:

Reply:
![Screenshot from 2023-06-28 15-23-39](https://github.com/binary-com/deriv-com/assets/93753441/b3620ddf-00ee-4347-90b8-8cb4281ed251)
@NikitK-deriv by adding that code we will face this issue.
Let's have a call later to decided on the best approach for this. Thank you.